docs/BackupRDB.md

```diff
@@ -0,0 +1,14 @@
+# Backup RDB guide
+
+This guide explains how to backup single RDB data with Scalar products on a Kubernetes environment. We assume that you use managed databases provided by cloud service.
+
+Even if you use RDB, you must follow the [Backup NoSQL database guide](./BackupNoSQL.md) when you use more than two RDBs under the Scalar products using [Multi-storage Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md) or [Two-phase Commit Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md).
```

Comment:
Previously, we mentioned "a single RDB." Should this sentence say "two or more RDBs"? If so, please see my suggestion below.

```suggestion
If you have two or more RDBs that are using the [Multi-storage Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/multi-storage-transactions.md) or [Two-phase Commit Transactions](https://github.com/scalar-labs/scalardb/blob/master/docs/two-phase-commit-transactions.md) feature, you must follow the instructions in [Back up a NoSQL database in a Kubernetes environment](./BackupNoSQL.md) instead.
```

Reply:
I feel `RDBs that are using the [Multi-storage Transactions] or [Two-phase Commit Transactions]` is `The database utilizes ScalarDB's feature`.
In other words, I feel that sentence means the ScalarDB is an extension for some databases like the following image.
```
            +---> [Database (with Multi-storage Transactions of ScalarDB)]
            |
[Client] ---+
            |
            +---> [Database (with Multi-storage Transactions of ScalarDB)]
```

However, in fact, the system architecture is the following.
```
                                                         +---> [Backend DB]
                                                         |
[Client] ---> [ScalarDB (Multi-storage Transactions)] ---+
                                                         |
                                                         +---> [Backend DB]
```

The ScalarDB utilize the several databases as `Backend DBs` to store some data of ScalarDB.
So, I think it would be better to describe `RDBs that are used by the [Multi-storage Transactions] or [Two-phase Commit Transactions]` instead of `RDBs that are using the [Multi-storage Transactions] or [Two-phase Commit Transactions]`.

What do you think?